omnibus/config/software/chef-workstation-app.rb

```diff
@@ -20,7 +20,7 @@
 license_file "LICENSE"
 
 source git: "https://github.com/chef/chef-workstation-app"
-default_version "master"
+default_version "SHACK-354/omnitruck-update-checks"
```

Comment:
this'll be reverted once the branch merges

Reply:
Can we get a revert pr queued up.

Reply:
I was just going to interactive rebase/omit this commit rather than having a temp change in the merged history

Reply:
And done, the branch is ready to go after the other PR merges.